### ISSUE TITLE
Default binding issue

### ISSUE THREAD

**Issue by NoMoreViolence** [inorganik/countUp.js-angular2#24]
```
[ ] Bug
[ ] Build issue
[V] Feature request
```

#### Version info 

```
Angular: 7.2.12
Angular CLI: 7.3.8
countup.js-angular2: 7.1.0
```

#### Description

I need default binding. 
For example, I create number type (100) class property, name is A.
then, I use the A to countup's endVal,
but it doesn't work.
This works with the endVal changed.

**Comment by kahnchen1028** [comment]
Hello @NoMoreViolence 
Maybe you can try this way to put the default value.  
  <span [countUp]="score" [options]="options" [reanimateOnClick]="false"
      >{{ score }}
    </span>

**Comment by NoMoreViolence** [comment]
But, I want animate default value too.

**Comment by inorganik** [comment]
Should be fixed now, I pushed 7.1.1. Please close if the update fixes your issue.
